src/GoMath.ts

```diff
@@ -211,9 +211,29 @@ export class GoMath {
         }
         return ret;
     }
+    public static readonly COOR_SEQ = "abcdefghijklmnopqrstuvwxyz";
+
+    public static coor_ch2num(ch: string): number {
+        return GoMath.COOR_SEQ.indexOf(ch?.toLowerCase());
+    }
+
+    public static coor_num2ch(coor: number): string {
+        return GoMath.COOR_SEQ[coor];
+    }
+
+    public static readonly PRETTY_COOR_SEQ = "abcdefghjklmnopqrstuvwxyz";
```

Comment:
I wonder if this PRETTY_COOR_SEQ should just be uppercase.  Both instances of `pretty_coor_num2ch` is followed by a .toUpperCase(), so I think the assumption is that pretty == uppercase (and no `I`).

Reply:
the feeling here was for both char seqs to be the same kind of "source of the truth", and upper/lowercase is just output format, but whatever & we will save an uppercase call & simplify, so dunno; no strong opinion

Reply:
fixed to your tip, no need to bikeshed here, it is a detail

Reply:
Indeed, I definitely should have labeled this a "nit". Anyway thanks for addressing!